documentation/src/main/asciidoc/cassandraclient.adoc

```diff
@@ -29,14 +29,14 @@ The elements are stored in the Cassandra database.
 We recommend that you follow the instructions in the next sections and create the application step by step.
 However, you can go right to the completed example.
 
-The solution is located in the `quickstart` {quickstart-clone-url}[directory].
+The solution is located in the `quickstart` link:https://github.com/datastax/cassandra-quarkus/tree/master/quickstart[directory].
```

Comment:
I propose to inline all used URLs to not use maven plugin for generating the output file.
Thx to that, we can leverage the github view:
https://github.com/datastax/cassandra-quarkus/blob/a2dc10c0975b0d05ed9c74471c69287a4dbe3838/documentation/src/main/asciidoc/cassandraclient.adoc
of the adoc file and point users to this file and point users to this file:
https://github.com/datastax/cassandra-quarkus/pull/66/files#diff-04c6e90faac2675aa89e2176d2eec7d8R39
(at least until the final adoc will be hosted in the final destination)

Reply:
Yes, that's great. I even wonder if we shouldn't go with asciidoc everywhere, including for these README files, for consistency with the documentation already present. Also asciidoc is more flexible, which could be useful e.g. to include code snippets in doc pages.

Reply:
I think that GitHub requires the MD format for CONTRIBUTING so I think we should leave it as MD.

Reply:
I don't think there is such a constraint, e.g. Spring projects use asciidoc for everything: https://github.com/spring-projects/spring-boot/blob/master/CONTRIBUTING.adoc.

Reply:
Ok, if you think that it should be ascidoc I can rework it. @polandll  any preference, suggestions with format should we pick (ascidoc or MD)?

Reply:
Definitely asciidoc if there is a choice. Much more flexible.